README.md

```diff
@@ -112,6 +112,7 @@ The following are the environment variables you have to configure to run a priva
 - `GITHUB_USER`: For CLA assistant to comment on pull requests, it requires a GitHub account.
 - `GITHUB_PASS`: Use CLA assistant's user and password for basic authentication (where github_token doesn't work).
 - `GITHUB_TOKEN`: Use GitHub token of CLA assistant's user for API calls of not authenticated users. It can be generated here https://github.com/settings/tokens/new. The Only scope required is `repo_public`.
+- `GITHUB_ADMIN_USER`: (optional), if set, will only allow the specified GitHub user to administer this instance of the app.
```

Comment:
Good idea! I like it.
I think it would make more sense if I could define a list of users who has admin rights.